pkg/logs/processor/processor.go

```diff
@@ -113,6 +119,13 @@ func (p *Processor) run() {
 				log.Errorf("Error while reconfiguring the SDS scanner: %v", err)
 				order.ResponseChan <- err
 			} else {
+			    p.flareCtl.SetStandardSDSRules([]string{})
+			    p.flareCtl.SetEnabledSDSRules([]string{})
```

Comment:
Since Set functions actually append, this is a no-op?

This is unlikely, but this can make SDS configuration appear empty in the flare when it is in fact not, and it would be very confusing in a support case. Would it make sense to clear only when SDS is not ready?

Reply:
The flare builder is testing for the rules count
```
	if len(fc.sdsInfos.standardRulesNames) > 0 || len(fc.sdsInfos.enabledRulesNames) > 0 {
```

About the readiness, my take is that if it's not ready, the logs are not processed: it better reflects the processing status. However it means `IsReady` might be slightly misnamed then 😅